### PR TITLE
- General editorial changes and improvements.

### PR DESCRIPTION
**Pull request by rjschwei**

### CODE REVIEW

**Comment by rjschwei** [comment]
@japelbaum @mfriesenegger Finally :D 

Please review/fix/suggest further improvements as you see fit.

@chabowski probably needs some help with tagging. Much appreciated

File: xml/MAIN-SBP-CSP-UpdateInfra.xml [hunk 1]
```diff
@@ -879,24 +900,57 @@ sudo chown -R _rmt:nginx /var/lib/rmt/public/repo
...
+                                        <!-- RJS -->
 					<screen>
-class InstanceVerification::Providers::Example &lt; InstanceVerification::ProviderBase
-	def instance_valid?
-		@instance_id = 'i-0000000000'
-		@instance_billing_info = 'example'
-		true # this is an example, always returns true
-	end
-end
+                                          class InstanceVerification::Providers::Example &lt; InstanceVerification::ProviderBase
```
Comment (rjschwei) [inline xml/MAIN-SBP-CSP-UpdateInfra.xml:920]: @jesusbv Can you please review the example code starting here and if this looks good update the provided example.rb script accordingly? Thanks

File: xml/MAIN-SBP-CSP-UpdateInfra.xml [hunk 2]
```diff
@@ -906,24 +960,38 @@ end
...
 					<itemizedlist>
-						<listitem>
+					        <listitem>
+							<para><emphasis>api_secret</emphasis> is a secret token only shared
+							between the RMT servers in the region. The same token must be configured
+                                                        on all servers that are expected to communicate with each other.</para>
+						</listitem>
+                                                <listitem>
+						  <para><emphasis>data_dir</emphasis> a directory where the registration
+                                                  sharing replay log is writte. The log is processed by the process contrilled by the
```
Comment (jesusbv) [inline xml/MAIN-SBP-CSP-UpdateInfra.xml:983]: s/writte/written

s/contrilled/controlled

File: xml/MAIN-SBP-CSP-UpdateInfra.xml [hunk 3]
```diff
@@ -1004,17 +1072,16 @@ systemctl enable rmt-server-regsharing.timer
 
 			<para>Region servers are located within a CSP network. An exmaple of the region server
```
Comment (jesusbv) [inline xml/MAIN-SBP-CSP-UpdateInfra.xml:1073]: s/exmaple/example

File: xml/MAIN-SBP-CSP-UpdateInfra.xml [hunk 4]
```diff
@@ -1244,28 +1317,86 @@ systemctl enable rmt-server-regsharing.timer
 
 			<itemizedlist>
 				<listitem>
-					<para>Place the public key file that was generated from the
-							<command>genRegionServerCert</command> command in the
+					<para>The public key files for all region servers generated with the
+					<command>genRegionServerCert</command> command. Filenames must be in pem format
+                                        and end with the .pem extension. further the names must match the names set in the configuration
```
Comment (jesusbv) [inline xml/MAIN-SBP-CSP-UpdateInfra.xml:1322]: s/further/Further or s/extension./extension,

File: xml/MAIN-SBP-CSP-UpdateInfra.xml [hunk 5]
```diff
@@ -1244,28 +1317,86 @@ systemctl enable rmt-server-regsharing.timer
...
+                          <listitem><option>server</option>
+                              <para>This section is processed by the client to obtain the necessary data to request update server information.
+                              The <emphasis role="strong">server</emphasis> supports the following options:
+                              <itemizedlist>
+			        <listitem>
+                                  <option>api</option>
+                                  <para>Specifies the API to use on the region server. This should be set to <value>regionInfo</value>
+                                  unless you are not using the regionserver code from the <link
+                                  xlink:href="https://github.com/SUSE-Enceladus/cloud-regionsrv">cloud-regionsrv</link> project or you have a
+                                  customized version of the code running server side that supports a different API. This option is mandatory.
```
Comment (jesusbv) [inline xml/MAIN-SBP-CSP-UpdateInfra.xml:1343]: s/mandatory./mandatory

File: xml/MAIN-SBP-CSP-UpdateInfra.xml [hunk 6]
```diff
@@ -1244,28 +1317,86 @@ systemctl enable rmt-server-regsharing.timer
...
+                                <listitem>
+                                  <option>certLocation</option>
+                                  <para>The fully qualified directory path where the region server certs are located. This must match
+                                  with the placement of the cerificates in the filesystem.</para>
+                                </listitem>
+                                <listitem>
+                                  <option>metadata_server</option>
+                                  <para>The URL for a metatdata server that provides the update server information in the expected format.
+                                  If the metadata server uses the https protocol the certificate of the server must be publicly signed.
+                                  It is expected that the specified metadata server if part of the framework infrastrcuture and as such is
```
Comment (jesusbv) [inline xml/MAIN-SBP-CSP-UpdateInfra.xml:1362]: s/infrastrcuture/infrastructure

File: xml/MAIN-SBP-CSP-UpdateInfra.xml [hunk 7]
```diff
@@ -1244,28 +1317,86 @@ systemctl enable rmt-server-regsharing.timer
...
+                            <itemizedlist>
+                              <listitem>
+                                <option>verifyAccess</option>
+                                <para>Reserved option name not used.</para>
+                              </listitem>
+                            </itemizedlist>
+                          </listitem>
+                        </itemizedlist>
+                        <para>The registration with the update infrastructure is set up by enabling the <package>guestregister</package>
+                        service in the image. This will enure an instance gets register upon first boot.</para>
```
Comment (jesusbv) [inline xml/MAIN-SBP-CSP-UpdateInfra.xml:1399]: s/enure/ensure

**Comment by chabowski** [comment]
> @japelbaum @mfriesenegger Finally :D
> 
> Please review/fix/suggest further improvements as you see fit.
> 
> @chabowski probably needs some help with tagging. Much appreciated

Hi @rjschwei I will do an editorial review (including tags) after the tech content is reviewed and changes are included, okay? (already realized that Travis CI fails, means some tagging might indeed be broken/wrong). Thanks!

File: xml/MAIN-SBP-CSP-UpdateInfra.xml (hunk 2 above)
Comment (sknorr) [reply xml/MAIN-SBP-CSP-UpdateInfra.xml:983]:
```suggestion
                                                  sharing replay log is written. The log is processed by the process controlled by the
```

File: xml/MAIN-SBP-CSP-UpdateInfra.xml (hunk 3 above)
Comment (sknorr) [reply xml/MAIN-SBP-CSP-UpdateInfra.xml:1073]:
```suggestion
			<para>Region servers are located within a CSP network. An example of the region server
```

File: xml/MAIN-SBP-CSP-UpdateInfra.xml [hunk 8]
```diff
@@ -879,24 +900,57 @@ sudo chown -R _rmt:nginx /var/lib/rmt/public/repo
...
+                                              def instance_valid?
+                                                  # Extract the instance identifier from the instance data sent by the client
+		                                  instance_product_id = verify_instance_data(@instance_data)
+                                                  return true if (@product_hash[:identifier].downcase == 'sles' && instance_product_id == SLES_PRODUCT_IDENTIFIER)
+                                                  return true if (@product_hash[:identifier].downcase == 'sles_sap' && instance_product_id == SLES4SAP_PRODUCT_IDENTIFIER)
+                                              
+                                                  raise InstanceVerification::Exception, 'Product/instance type mismatch'
+                                              end
+
+                                              def verify_instance_data(instance_data)
```
Comment (jesusbv) [inline xml/MAIN-SBP-CSP-UpdateInfra.xml:935]: s/verify/validate

File: xml/MAIN-SBP-CSP-UpdateInfra.xml [hunk 9]
```diff
@@ -879,24 +900,57 @@ sudo chown -R _rmt:nginx /var/lib/rmt/public/repo
...
-end
+                                          class InstanceVerification::Providers::Example &lt; InstanceVerification::ProviderBase
+                                              # Unique identifier for SLES sent by client instance
+                                              SLES_PRODUCT_IDENTIFIER = 1234_SUSE_SLES.freeze
+                                              # Unique identifier for SLES For SAP sent by client instance
+                                              SLES4SAP_PRODUCT_IDENTIFIER = 6789_SUSE_SAP.freeze
+                                              
+                                              def instance_valid?
+                                                  # Extract the instance identifier from the instance data sent by the client
+		                                  instance_product_id = verify_instance_data(@instance_data)
```
Comment (jesusbv) [inline xml/MAIN-SBP-CSP-UpdateInfra.xml:928]: s/verify/validate

File: xml/MAIN-SBP-CSP-UpdateInfra.xml (hunk 1 above)
Comment (jesusbv) [reply xml/MAIN-SBP-CSP-UpdateInfra.xml:920]: Done

**Comment by chabowski** [comment]
@jesusbv @japelbaum @mfriesenegger All proposed changes are already committed to the PR. Jona, Mike, if you don't have edits to the PR from Robert, I can merge the PR as soon as you want. thanks very much!

**Review by lproven — CHANGES_REQUESTED** [review]
General comment from the team editor. :-)

We have a "house style" for DocBook XML formatting, which includes various rules such as "indent by 1 space only" and "tags go on their own lines, except [screen]" and so on.

Because there are several of them, we have an automated tool for applying them. It is part of the DAPS suite.

You can either use:

`daps-xmlformat -i FILE`

or: 

`daps -d DC-FILE xmlformat` 

This will automatically reformat the specified file in house style.

I suggest doing this as it will make this doc much easier to read...

**Review by japelbaum — APPROVED** [review]

**Comment by chabowski** [comment]
> General comment from the team editor. :-)
> 
> We have a "house style" for DocBook XML formatting, which includes various rules such as "indent by 1 space only" and "tags go on their own lines, except [screen]" and so on.
> 
> Because there are several of them, we have an automated tool for applying them. It is part of the DAPS suite.
> 
> You can either use:
> 
> `daps-xmlformat -i FILE`
> 
> or:
> 
> `daps -d DC-FILE xmlformat`
> 
> This will automatically reformat the specified file in house style.
> 
> I suggest doing this as it will make this doc much easier to read...

Hi @lproven Liam - I will care for it, thanks!

**Pull request merged**